assets/styles/base/_typography.scss

```diff
@@ -37,10 +37,43 @@ H6 {
 }
 
 P {
-  font-size: 1.4em;
+  // font-size: 1.4em;
   line-height: 1.75em;
```

Comment:
I don't know why `<p>` had a huge font to begin with, but shouldn't the line-height be removed too?